src/transformState.js

```diff
@@ -5,7 +5,20 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+      case 'clear':
+        Object.keys(state).forEach(key => delete state[key]);
+    }
```

Comment:
```suggestion
        break;
        
      default:
        break;
    }
```

It is considered good practice to always include a "default" case.